### DISCUSSION TITLE
Cannot start a laravel shadcn

### DISCUSSION THREAD

**Discussion by Lorh-o** [shadcn-ui/ui#4850] [General]
laravel new my-app --typescript --breeze --stack=react --git --no-interaction

Using this command, I get the return "--typescript" option does not exist.. I have been trying to get a laravel & react project setup for the longest time, and now that I find these docs that explain how to start one with shadcn. The incorrect commands are given, does anyone have better instructions on how to start a shadcn laravel project that doesn't have any issues?

**Comment by Lorh-o** [comment]
Following https://ui.shadcn.com/docs/installation/laravel

**Reply by mrlinnth** [reply]
I just faced same issue. I had no issue when I was testing shadcn with laravel on my mac. But when I tried to run a new project on linux, I faced same issue.

I think issued is not related to shadcn and more of laravel/installer. 

The --typescript option is available for laravel/installer 5 and above I guess. My linux laravel/installer package is at version 4.5.
